torch/quantization/fx/quantize.py

```diff
@@ -23,18 +22,18 @@
 
 from .pattern_utils import (
     matches,
-    register_quant_pattern,
     get_quant_patterns,
-    register_dynamic_pattern,
     get_dynamic_quant_patterns,
 )
 
 from .utils import _parent_name
 
-from abc import ABC, abstractmethod
+# TODO before land: discuss is we want to namespace this instead,
+#   something like patterns.CopyNode instead of CopyNode
+from .patterns import *
```

Comment:
usually `import *` is not ideal, this could be the rare case where it's acceptable.  Or, we could just prefix everything with `patterns.FooPattern`.  lmk what people prefer

Reply:
I think we can explicitly import CopyNode, looks like it's the only one used here. also patterns should probably be merged with pattern_utils?

Reply:
> patterns should probably be merged with pattern_utils?

IMO it's a bit confusing to have files with large LOC even if they are about a single thing.  Cleanly splitting them makes it easier to read for people who didn't write the code.

Reply:
sure, are you suggesting to split dynamic patterns to a different file as well?

Reply:
why do we want to split pattern_utils and patterns in this case actually?